apps/arweave/src/ar_mining_io.erl

```diff
@@ -122,6 +127,15 @@ handle_call({read_recall_range, WhichChunk, Worker, Candidate, RecallRangeStart}
 	end,
 	{reply, ThreadFound, State};
 
+handle_call({is_recall_range_readable, Candidate, RecallRangeStart}, _From, State) ->
```

Comment:
If possible would be good to dedupe the duplicate code between this handler and `read_recall_range`